test/new_tests/test_zserializers.py

```diff
@@ -215,24 +179,6 @@ def test_with_class_serializer_and_instance_serializer(self):
         assert bins == {"normal": 1234, "tuple": ("instance serialized", "instance deserialized")}
         client.close()
 
-    def test_with_unset_serializer_python_serializer(self):
-        """
-        Invoke put() for mixed data record with python serializer and
-        calling unset_serializers
-        """
-        aerospike.set_serializer(json.dumps)
-        aerospike.set_deserializer(json.loads)
-        method_config = {}
-        client = TestBaseClass.get_new_connection(method_config)
-
-        aerospike.unset_serializers()
-        client.put(self.test_key, self.mixed_record, serializer=aerospike.SERIALIZER_PYTHON)
```

Comment:
`SERIALIZER_PYTHON` is still in the serializer code. Should some test cases for it be kept?

Reply:
I left the constant in because a lot of code for expressions and `client.operate()` operations still uses helper functions with SERIALIZER_PYTHON hardcoded as an argument, and those helper functions use SERIALIZER_PYTHON to convert bytes and bytearrays in expressions and operations to the C client as_bytes.

https://github.com/aerospike/aerospike-client-python/blob/CLIENT-2258-rm-auto-serialize/src/main/serializer.c#L325-L335

If it makes things easier to understand, I can try to refactor the code so the code uses SERIALIZER_NONE by default.

Reply:
Ok, thanks for the explanation. Sounds ok to me.